includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -699,6 +699,19 @@ private function validate_attr_spec_list_for_node( $node, $attr_spec_list ) {
 				}
 			}
 
+			/*
+			 * If given attribute's value is a URL with a host, the host must
+			 * be valid
+			 */
+			if ( isset( $attr_spec_rule[ AMP_Rule_Spec::VALUE_URL ][ AMP_Rule_Spec::VALID_HOST ] ) ) {
```

Comment:
Instead of introducing `AMP_Rule_Spec::VALID_HOST` here, which isn't actually part of the spec, I suggest you validate the host whenever `isset( $attr_spec_rule[ AMP_Rule_Spec::VALUE_URL ] )`. Should it not be validated for every attribute that is a URL value?